### PR TITLE
Switch to GH Actions, fix tests, update network libraries

### LINKED ISSUE #49: migrate CI to github actions
Build is stalled: https://github.com/georust/geocoding/pull/48#issuecomment-896745354

Screen Shot 2021-08-11 at 9 35 59 AM

It seems like maybe bors is set up OK and has appropriate access, but the build status never updates.

I think this is because this repo still relies on travis-ci. 

There's too much verbage here for me to be willing to read right now: https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

But I believe the take away is that travis-ci is no longer an option for us unless we want to pay, so I'd recommend switching to github actions instead of travis, like we have for most of our other repositories.

### PR DESCRIPTION
**Pull request by urschrei**
Fixes #49

### CODE REVIEW

**Comment by michaelkirk** [comment]
lgtm!

bors r+

**Comment by urschrei** [comment]
🤦‍♂️ added you @michaelkirk

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "geocoding"
 description = "Geocoding library for Rust"
 version = "0.3.1"
-authors = ["Stephan Hügel <urschrei@gmail.com>", "Blake Grotewold <hello@grotewold.me>"]
+authors = ["The Georust Developers <admins@georust.org>"]
```
Comment (michaelkirk) [inline Cargo.toml:5]: I just tried to contact this address and it bounced:

```
<admins@georust.org>: host gmr-smtp-in.l.google.com[2607:f8b0:400d:c0f::e]
   said: 550-5.1.1 The email account that you tried to reach does not exist.
   Please try 550-5.1.1 double-checking the recipient's email address for
   typos or 550-5.1.1 unnecessary spaces. Learn more at 550 5.1.1
   https://support.google.com/mail/?p=NoSuchUser b14si109472pjg.2 - gsmtp (in
   reply to RCPT TO command)
```

I understand why we might not want to have individual authors listed, but putting in a fake email address seems fraught.

Comment (urschrei) [reply Cargo.toml:5]: It _should_ be working…

Comment (michaelkirk) [reply Cargo.toml:5]: see https://github.com/georust/meta/issues/28

**Pull request merged**